jest.vrt.config.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+/*eslint-env node*/
+
+module.exports = {
+  testPathIgnorePatterns: ['/node_modules/', '/dist/', './babel/cup.js'],
+  modulePathIgnorePatterns: ['<rootDir>/dist/'],
+  preset: 'jest-puppeteer',
+  testRunner: 'jest-circus/runner',
+  testRegex: 'vrt.js$',
```

Comment:
are you planning to change this to `scenario.js`? that's what screener is using currently

Reply:
The `vrt.js` files are intended for specifying jest unit tests vs the actual storybook scenarios.

Reply:
Could this be collapsed into the `.e2e.js` tests? What benefit do we gain from running the vrts separately? Both rely on the static server infrastructure. I'm not particularly excited about adding another file if it's not strictly necessary

Reply:
i think this is mostly resolved by moving all tests to the top level - all scenarios are tested by default and you only need to specify interactions.

moving the interactions to e2e tests worries me a bit due to the reliance on docker for visual consistency. local test runs would require running in docker, at least to avoid snapshots being flagged/updated. it may not be that bad in practice though, so worth a try.

Reply:
fine with me, in this case. what I most worry about is having code in one part of the codebase, that is not near dependent code

Reply:
i've added a future todo for this.

either move the interactions into e2e or into scenario files as exports. i kind of like the latter pattern, which is similar to the screener steps api thing, and has the best co-location. i think this means adding a build step for vrt to access exports from es modules.

for now there is a top level config object that is kinda ugly but not terrible.